frontend/src/pages/ProductDisplay.jsx

```
@@ -1,71 +1,92 @@
+              <Button className="w-fit">
+                <button
+                  onClick={() =>
+                    dispatch(
+                      postToCart({
+                        userId: user?.id,
+                        productId: productInfo.id,
+                        productPrice: productInfo.price,
+                      })
+                    )
+                  }
+                >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)